packages/gatsby-cli/src/index.ts

```diff
@@ -17,17 +17,29 @@ if (useJsonLogger) {
 // Check if update is available
 updateNotifier({ pkg }).notify({ isGlobal: true })
 
-const MIN_NODE_VERSION = `>=8.0.0`
+const MIN_NODE_VERSION = `8.0.0`
 
-if (!semver.satisfies(process.version, MIN_NODE_VERSION)) {
+if (!semver.satisfies(process.version, `>=${MIN_NODE_VERSION}`)) {
   report.panic(
     report.stripIndent(`
-      Gatsby requires Node.js v8 or higher (you have ${process.version}).
+      Gatsby requires Node.js ${MIN_NODE_VERSION} or higher (you have ${process.version}).
       Upgrade Node to the latest stable release: https://gatsby.dev/upgrading-node-js
     `)
   )
 }
 
+const NEXT_MIN_NODE_VERSION = `10.13.0`
+
+if (!semver.satisfies(process.version, `>=${NEXT_MIN_NODE_VERSION}`)) {
+  report.warn(
+    report.stripIndent(`
+      Gatsby will be dropping support for Node.js ${process.version} soon 
+      and will only actively support ${NEXT_MIN_NODE_VERSION} or higher.
+      Please upgrade Node.js to a currently active LTS release: https://gatsby.dev/upgrading-node-js
```

Comment:
https://www.gatsbyjs.org/docs/upgrading-node-js/ that is linked in deprecation warning/notice specifically mentions updating from Node 6 to Node 8 (with concrete instructions on installing Node 8), this will need to be updated as well

Alternatively, we can ship warning without linking and update the doc separately (?)

Reply:
Great point! Updated the docs